config/default.schema.json

```diff
@@ -66,6 +66,11 @@
 			"default": false,
 			"description": "If enabled, firstLogin page is not shown."
 		},
+		"FEATURE_LOG_REQUEST": {
```

Comment:
Please enable it for development as default. For production default false is ok.

Reply:
fixed